src/resources/FlightPrice.php

```diff
@@ -5,8 +5,8 @@
 namespace Amadeus\Resources;
 
 /**
- * Sub-resource in FlightOffer, etc.
- * @see FlightOffer
+ * Sub-resource in FlightOffer, FlightDestination etc.
```

Comment:
So in this sub-resource, instead of creating a new resource `Price.php`, you use the `FlightPrice` that has in common the field `total`, like mentioned in the call earlier?

Reply:
Yes, and even the ```FlightPrice``` resource in the ```FlightOffer``` API reference is also named ```Price```, so actually they are the same name model. And my purpose is to reuse this ```FlightPrice``` for all the flight-related APIs if they contain the same name ```Price``` model.